### PR TITLE
Set EIGEN_VECTORIZE_SSE3 when building for windows using bazel.

### PR DESCRIPTION
**Pull request by gunan**

### CODE REVIEW

**Comment by gunan** [comment]
Running bazel presubmit here:
http://ci.tensorflow.org/job/tf-pr-win-bzl/5/
(I really should automate this one)

File: tensorflow/tensorflow.bzl [hunk 1]
```diff
@@ -111,6 +111,7 @@ def tf_copts():
                 "/DPLATFORM_WINDOWS",
                 "/DEIGEN_HAS_C99_MATH",
                 "/DTENSORFLOW_USE_EIGEN_THREADPOOL",
+		"/DEIGEN_VECTORIZE_SSE3",
```
Comment (girving) [inline tensorflow/tensorflow.bzl]: Add `// To flush denormals without __SSE3__ set`.  Terrifyingly obscure otherwise.

**Comment by girving** [comment]
Does `-ftz=true` already get set for Windows Bazel?

**Comment by mrry** [comment]
@girving Looks like it does get set: https://github.com/tensorflow/tensorflow/blob/e60e72435f0dfebe6424ab4c525523486006d47a/tensorflow/tensorflow.bzl#L370

**Review by girving — APPROVED** [review]

**Comment by gunan** [comment]
http://ci.tensorflow.org/job/tf-pr-win-bzl/6/console
In bazel the test is now fixed.
Jenkins, test this please.

**Pull request merged**